### PR TITLE
Add attendance tracking column to permissions table

### PR DESCRIPTION
**Pull request by zandre-eng**
## Product Description

Small UI change that adds a new column in the "Roles and Permissions" page to show which roles have the `manage_attendance_tracking` permission enabled.

Screenshot from 2023-04-13 16-58-30

## Technical Summary

Link to ticket [here](https://dimagi-dev.atlassian.net/browse/SC-2686).

## Feature Flag

`ATTENDANCE_TRACKING`

### Safety story

Small UI change, have done local testing.

### Automated test coverage

N/A

### QA Plan

No QA planned.

### Rollback instructions

- [X] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [X] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/users/templates/users/roles_and_permissions.html [hunk 1]
```diff
@@ -123,6 +123,11 @@ <h3 class="panel-title">
             {% trans "Data Registries" %}
           </th>
           {% endif %}
+          {% if request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```
Comment (kaapstorm) [inline corehq/apps/users/templates/users/roles_and_permissions.html:126]: I think Attendance Tracking has both a privilege and a (temporary) toggle associated with it.
```suggestion
          {% if request|request_has_privilege:"ATTENDANCE_TRACKING" and request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```

File: corehq/apps/users/templates/users/roles_and_permissions.html [hunk 2]
```diff
@@ -239,6 +244,14 @@ <h6 data-bind="visible: !name()">({% trans "No Name" %})</h6>
             </div>
           </td>
           {% endif %}
+          {% if request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```
Comment (kaapstorm) [inline corehq/apps/users/templates/users/roles_and_permissions.html:247]: Ditto
```suggestion
          {% if request|request_has_privilege:"ATTENDANCE_TRACKING" and request|toggle_enabled:"ATTENDANCE_TRACKING" %}
```

Comment (Charl1996) [reply corehq/apps/users/templates/users/roles_and_permissions.html:247]: Unrelated, but after a google search on the word "Ditto" I realised I've been using it all wrong my whole life!

**Review by Charl1996 — APPROVED** [review]

**Comment by Charl1996** [comment]
Good to merge once Norman's concerns are addressed.

**Comment by zandre-eng** [comment]
Privilege check added in with toggle check [ce7efae](https://github.com/dimagi/commcare-hq/pull/32835/commits/ce7efae794f12f5732f8885a578093b6714ffb1e).

**Pull request merged**